sites/hashdev/src/pages/_app.page.tsx

```diff
@@ -5,12 +5,28 @@ import "../../styles/prism.css";
 import { EmotionCache } from "@emotion/react";
 import type { AppProps } from "next/app";
 import { useRouter } from "next/router";
+import { DefaultSeo, DefaultSeoProps } from "next-seo";
 import { useEffect, FunctionComponent } from "react";
 import NextNProgress from "nextjs-progressbar";
 import { PageLayout } from "../components/PageLayout";
 import { theme } from "../theme";
 import { MuiProvider } from "../theme/MuiProvider";
 import { NextPageWithLayout } from "../util/nextTypes";
+import { FRONTEND_URL, SITE_DESCRIPTION } from "../config";
+
+const defaultSeoProps: DefaultSeoProps = {
+  title: "HASH.dev – HASH for Developers",
+  description: SITE_DESCRIPTION,
+  twitter: {
+    cardType: "summary_large_image",
+    site: "@hashintel",
```

Comment:
We can use `@hashdevs` here

Reply:
Thanks - updated in https://github.com/hashintel/hash/pull/1487/commits/9e1e9217287ea9bcde0d0d193f03da90e177845a